x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts

```diff
@@ -310,6 +311,18 @@ export class SpacesSavedObjectsClient implements SavedObjectsClientContract {
     });
   }
 
+  public async bulkResolve<T = unknown>(
```

Comment:
nit: add JSDocs for consistency within this file

Reply:
I meant to comment on this but forgot, so I'm glad you brought it up!

I don't really like how we copy/pasted the TSDocs for every function in the Spaces SOC wrapper (and there has been some drift over time, so they aren't all in sync with the base SOC TSDocs). In addition, we don't have these comments for the Security or ESO SOC wrappers.

In my opinion we should remove all of the TSDocs from the Space SOC wrapper. I planned to do that in a follow up but I'll just do it in this PR unless you feel strongly about it

Reply:
🤷‍♂️ I don't feel strongly here

Reply:
OK, I'll remove the rest of the redundant TSDocs